proto/nodenetworkservice/3.302.0.744/NodeNetworkService.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+package azure.nodenetworkservice;
+option go_package = ".;nodenetworkservice";
+
+service NodeNetworkService {
+    rpc ConfigureNetworking(ConfigureNetworkingRequest) returns (ConfigureNetworkingResponse);
```

Comment:
can you add short description on what these api does?

also whats the diff between ConfigureNetworking and ConfigureContainerNetworking

Reply:
Again, this file comes from nns team. in future it could come via a nuget too. So we don't want to edit it. I have already added description of ConfigureContainerNetwoking in grpc client. Other APIs for other services that consume this contract. We only consume ConfigureContainerNetworking API.